### ISSUE TITLE
[Adaptive Cards- Import- Expense Report]: 'ER-132' control is not accessible with keyboard using Tab key on Expense Report Pane.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6974]
https://microsoft.visualstudio.com/OS/_workitems/edit/34055892
